src/renderer/frontend.ts

```diff
@@ -0,0 +1,81 @@
+import { FrontendBase, Scope } from '@sentry/core';
+import { Breadcrumb, Context, SdkInfo, SentryEvent } from '@sentry/shim';
+// tslint:disable-next-line:no-implicit-dependencies
+import { ipcRenderer } from 'electron';
+import {
+  CommonFrontend,
+  ElectronOptions,
+  IPC_CONTEXT,
+  IPC_CRUMB,
+  IPC_EVENT,
+} from '../common';
+import { RendererBackend } from './backend';
+
+/** Frontend implementation for Electron renderer backends. */
+export class RendererFrontend
+  extends FrontendBase<RendererBackend, ElectronOptions>
+  implements CommonFrontend {
+  /**
+   * Creates a new Electron SDK instance.
+   * @param options Configuration options for this SDK.
+   */
+  public constructor(options: ElectronOptions) {
+    super(RendererBackend, options);
+  }
+
+  /**
+   * @inheritDoc
+   */
+  protected getSdkInfo(): SdkInfo {
+    return {};
+  }
+
+  /**
+   * @inheritDoc
+   */
+  public getInitialScope(): Scope {
+    return {
+      breadcrumbs: [],
+      context: {},
+    };
+  }
+
+  /**
+   * Uploads a native crash dump (Minidump) to Sentry.
+   *
+   * @param path The relative or absolute path to the minidump.
+   * @param event Optional event payload to attach to the minidump.
+   * @param scope The SDK scope used to upload.
+   */
+  public async captureMinidump(
+    _path: string,
+    _event: SentryEvent,
+    _scope: Scope,
+  ): Promise<void> {
+    // TODO: Figure out whether this makes sense
```

Comment:
Implementing this is required by the common interface, but it doesn't really make sense to have this in the renderer. The Electron crashReporter does not allow to create dumps on-the-fly and renderers _might_ not have access to the file system anyway. So the default way to handle this would be an IPC to the main process. 

Now, we can issue an IPC call here, but I'm questioning how useful that would really be. The other alternative would be to completely remove the `CommonFrontend` interface and instead have fully separate SDKs, e.g. `@sentry/electron/main` and `@sentry/electron/renderer`. That comes with a whole range of different problems, though.